dvc/remote/oss.py

```diff
@@ -119,3 +119,7 @@ def _download(
         self.oss_service.get_object_to_file(
             from_info.path, to_file, progress_callback=cb
         )
+
+    def exists(self, path_info):
+        paths = self._list_paths(path_info.bucket, path_info.path)
```

Comment:
_list_paths here doesn't accept 2 arguments :)

Reply:
Fixing

Reply:
And the tests didn't catch this because we are running oss tests only on master. It would be a suprise for later :D This "not-running-some-tests-on-PR" issue will be addressed later.

Reply:
Hmm, thats not good, do we have some task to make oss tests run during build? Or is there a reason why they run on master only?